packages/filebrowser-extension/src/index.ts

```diff
@@ -297,9 +297,19 @@ function activateBrowser(
       if (navigateToCurrentDirectory) {
         const { newValue } = change;
         const context = docManager.contextForWidget(newValue);
+        const { path } = context;
```

Comment:
This fails if the current widget is not a document, since the context in that case is `undefined`. It should be moved inside of the `if` statement.